content/md/supplier-onboarder/create-products.md

```diff
@@ -6,7 +6,7 @@ popular: true
 ---
 
 # Product creation? Piece of cake!
-
+## Step by step clicks
```

Comment:
Not sure of the expression.
Maybe something like: "Step by step via the UI" or "Manual creation"?

Reply:
go for "Manual creation"